karapace/protobuf/proto_normalizations.py

```diff
@@ -0,0 +1,220 @@
+"""
+Copyright (c) 2024 Aiven Ltd
+See LICENSE for details
+"""
+from karapace.protobuf.enum_constant_element import EnumConstantElement
+from karapace.protobuf.enum_element import EnumElement
+from karapace.protobuf.extend_element import ExtendElement
+from karapace.protobuf.field_element import FieldElement
+from karapace.protobuf.group_element import GroupElement
+from karapace.protobuf.message_element import MessageElement
+from karapace.protobuf.one_of_element import OneOfElement
+from karapace.protobuf.option_element import OptionElement
+from karapace.protobuf.proto_file_element import ProtoFileElement
+from karapace.protobuf.rpc_element import RpcElement
+from karapace.protobuf.service_element import ServiceElement
+from karapace.protobuf.type_element import TypeElement
+from karapace.typing import StrEnum
+from typing import List
+
+
+class ProtobufNormalisationOptions(StrEnum):
```

Comment:
Not used, I'd remove. I also want to challenge the need for different normalization options also in future. I think there should be only one normalized/canonical format that Karapace follows, this would make the output stable.